fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/bridge/facades/SimulationFacade.java

```diff
@@ -649,31 +644,25 @@ private VehicleSensorData rearDistance(double rearDistance) {
                 continue;
             }
             if (queryingAllVehiclesRequired || vehiclesWithFrontSensor.containsKey(veh.id)) {
-                try {
-                    LeadingVehicle leader = ((VehicleSubscriptionResult) veh).leadingVehicle != null
-                            ? ((VehicleSubscriptionResult) veh).leadingVehicle
-                            : vehicleGetLeader.execute(bridge, veh.id, Double.MAX_VALUE);
```

Comment:
This was the initial reason why I removed the command. Even when subscriptions for leading vehicle was active, the "GetLeader" command was always called if no leader was returned in the subscriptions, resulting in many unnecessary TraCI calls.